code/_onclick/item_attack.dm

```diff
@@ -56,7 +56,11 @@
 	if(force && HAS_TRAIT(user, TRAIT_PACIFISM))
 		to_chat(user, "<span class='warning'>You don't want to harm other living beings!</span>")
 		return
-
+		if(istype(user, /mob/living/carbon/human))
+			var/mob/living/carbon/human/H = user
+			if(istype(H.mind.martial_art, /datum/martial_art/monk))
+				to_chat(user, "<span class='warning'>Your vows as a Monk prevent you from using [src] in combat!</span>")

```

Comment:
make a trait

Reply:
Will do

Reply:
Think I did what you wanted but I'm not sure.

Reply:
Did some testing with the original stuff and this bit of code isn't working right now even before I messed with it but was on the original 2018 version. Will see if I can fix.

Reply:
Well I found the issue that broke it, apparently I didn't port this correctly at this part of the code but it works now. I however after fixing it have tried tying this with the monk trait however no matter what I did as soon as I added a 	if(HAS_TRAIT(H, MONK_TRAIT)) it would break again and wouldn't work. Unless you or someone else can figure out a way to get it to work I will consider this resolved.